corehq/motech/repeaters/tests/test_dbaccessors.py

```diff
@@ -184,3 +264,50 @@ def tearDownClass(cls):
 
     def test_get_domains_that_have_repeat_records(self):
         self.assertEqual(get_domains_that_have_repeat_records(), ['a', 'b', 'c'])
+
+
+class TestSQLDBAccessors(RepeaterTestCase):
+
+    def test_get_paged_sql_repeat_records_daterange(self):
+        domain = 'test-sql'
+        eve = self.repeater.repeat_records.create(
+            domain=domain,
+            payload_id='eve',
+            registered_at='1970-02-01',
+        )
+        moon = self.repeater.repeat_records.create(
+            domain=domain,
+            payload_id='moon',
+            registered_at='1970-02-01',
+        )
+        eve_old = SQLRepeatRecordAttempt.objects.create(
+            repeat_record=eve,
+            created_at=datetime.utcnow() - timedelta(days=2)
+        )
+        moon_new = SQLRepeatRecordAttempt.objects.create(
+            repeat_record=moon,
+            created_at=datetime.utcnow() + timedelta(days=2)
+        )
+        self.assertQuerysetEqual(
+            get_paged_sql_repeat_records(domain, 0, 10),
+            [eve, moon],
+            ordered=False
+        )
```

Comment:
would be nice to split these into individual tests

Reply:
I thought this will be more readable than splitting since it essentially tests a single logic.

Reply:
That looks great, thanks.